Add accomodations

jbelien opened the pull request.

marjanvandekauter-tomtom commented:
@jbelien 

- Typo in the title: "Accomodations" should be "Accommodations"
- Can you add the distances to BluePoint mentioned in the [issue](https://github.com/osmbe/sotm-eu-2023/issues/99)?
- I can't tell if the page one is directed to when clicking "Crowne Plaza Antwerp" is the same page the link in the [issue](https://github.com/osmbe/sotm-eu-2023/issues/99) points to ([this link](https://www.crowneplaza.com/redirect?path=hd&brandCode=CP&localeCode=en&regionCode=1&hotelCode=anrbe&_PMID=99801505&GPC=T10&cn=no&viewfullsite=true)). Can you double check?

Thanks!

jbelien commented:
> Typo in the title: "Accomodations" should be "Accommodations"

Fixed! ✅ 

> Can you add the distances to BluePoint mentioned in the https://github.com/osmbe/sotm-eu-2023/issues/99?

Fixed! ✅ 

> I can't tell if the page one is directed to when clicking "Crowne Plaza Antwerp" is the same page the link in the https://github.com/osmbe/sotm-eu-2023/issues/99 points to ([this link](https://www.crowneplaza.com/redirect?path=hd&brandCode=CP&localeCode=en&regionCode=1&hotelCode=anrbe&_PMID=99801505&GPC=T10&cn=no&viewfullsite=true)). Can you double check?

This is correct.

marjanvandekauter-tomtom reviewed (approved):
I checked the preview and I approve
